### PR TITLE
account settings

### PR DESCRIPTION
**Pull request by jkeussen**
continued from https://github.com/wunderlist/web-essentials/pull/11

### CODE REVIEW

File: src/css/lib/tables.css [hunk 1]
```diff
@@ -37,31 +37,15 @@ td {
 td { vertical-align: top }
 
 th,
-tr:nth-child(even) td {
-  background-color: var(--off-white);
+tbody tr:nth-child(2n) td {
```
Comment (mlabod) [inline src/css/lib/tables.css]: is this a bad merge or why did you change it back?

File: src/css/lib/tables.css [hunk 2]
```diff
@@ -37,31 +37,15 @@ td {
...
+  background-color: var(--off-white)
 }
 
 tr:last-child td {
   border-bottom: 0;
 }
 
-td:last-child,
-th:last-child {
+tr td:last-child,
```
Comment (mlabod) [inline src/css/lib/tables.css]: same here

File: src/css/lib/tables.css [hunk 3]
```diff
@@ -37,31 +37,15 @@ td {
...
+tr th:last-child {
   border-right: 0;
 }
-
```
Comment (mlabod) [inline src/css/lib/tables.css]: and all of this

File: src/css/lib/tables.css (hunk 1 above)
Comment (jkeussen) [inline src/css/lib/tables.css]: it is probably a bad merge, I was struggling with a bunch of conflicts. I can take a look at it and fix it - sorry!

File: src/css/lib/buttons.css [hunk 4]
```diff
@@ -47,7 +65,7 @@ button,
   padding-right: var(--space-1);
 }
 
-.button {
+.button, .small-button {
```
Comment (mlabod) [inline src/css/lib/buttons.css]: isn't that the opposite site of pattern the whole thing is trying to do?

File: src/css/lib/buttons.css [hunk 5]
```diff
@@ -27,6 +27,24 @@ button,
   letter-spacing: 0.07em;
 }
 
+.small-button {
+  position: relative;
```
Comment (mlabod) [inline src/css/lib/buttons.css]: why relative?

File: src/services-modal.html [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+        <p class="m0 py1 vm col-8 md-col col pt1 inline-block">kate.spencer@gmail.com</p>
+        <div class="col md-col col-4">
+          <button class="small-button inline-block button right mt1 p1 caps primary o0">Make Primary</button>
+          <span class="h1 o0 pictogram-icon wundercon red mr2 right pointer icon-trash" title="trash"></span>
+        </div>
+      </div>
+      <div class="remove col md-col py2 border-bottom col-12">
+        <p class="m0 py1 vm col-8 col md-col pt1 inline-block">kate.spencer@gmail.com</p>
+        <div class="col md-col col-4">
+          <button class="small-button inline-block button right mt1 primary p1 caps o0">Make Primary</button>
```
Comment (mlabod) [inline src/services-modal.html]: here you define ``display:inline-block`` with 3 classes, thats kind of messy :grin:

**Comment by mlabod** [comment]
can't you add the modal within the service example page, otherwise you duplicate the main html on 3 pages, right?

File: src/Services.html [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+        font-size: .65rem;
+      }
+
+      .fourteen{
+        font-size: 0.875em;
+        line-height: 1.35rem;
+      }
+
+
+      .h1.pictogram-icon.wundercon.red.mr2.icon-trash {
```
Comment (mlabod) [inline src/Services.html]: why are extra declarations in here? and why this complicated selector?

File: src/mail2wl-success.html [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+        font-size: .65rem;
+      }
+
+      .o0 { 
+        opacity: 0;
+      }
+
+      .remove:hover span, .remove:hover .primary {
+        opacity: 1;
+        -webkit-transition-duration: 0.3s;
```
Comment (mlabod) [inline src/mail2wl-success.html]: can be used without prefix, according to caniuse.com
